### PR TITLE
basic rhessi tests

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes https://github.com/sunpy/sunpy/issues/5595

Copy and paste of the sdo/aia ones just AIA to rhessi

### CODE REVIEW

File: sunpy/map/sources/tests/test_rhessi_source.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+from sunpy.map import Map
+from sunpy.map.sources.rhessi import RHESSIMap
+from sunpy.util.exceptions import SunpyMetadataWarning
+
+params = [get_test_filepath("hsi_image_20101016_191218.fits")]
+
+
+@pytest.fixture(scope="module", params=params)
+def rhessi_map(request):
+    return Map(request.param)
```
Comment (wtbarnes) [inline sunpy/map/sources/tests/test_rhessi_source.py:16]:
```suggestion
@pytest.fixture(scope="module")
def rhessi_map():
    return Map(get_test_filepath("hsi_image_20101016_191218.fits"))
```

**Review by wtbarnes — APPROVED** [review]
Just one small suggestion to simplify the fixture.

**Review by dstansby — APPROVED** [review]
Nice - feel free to self merge when pre-commit is fixed.

**Comment by Cadair** [comment]
pre-commit.ci autofix

**Pull request merged**